sdk/identity/identity/test/authTestUtils.ts

```diff
@@ -76,10 +77,19 @@ export class MockAuthHttpClient implements HttpClient {
       throw new Error("The number of requests has exceeded the number of authResponses");
     }
 
+    const authResponse = this.authResponses[this.currentResponse];
+
+    if (authResponse.error) {
+      this.currentResponse++;
```

Comment:
Totally fine to leave it, since it's existing code, but  I think `currentResponseId` or `currentResponseIndex` would have been more descriptive

Reply:
I'll change it!